image/templates/helm/stackrox-central/internal/defaults.yaml.htpl

```diff
@@ -80,7 +80,17 @@ defaults:
         maxConns: 90
         statementTimeoutMs: 1200000
 
-      postgresConfig: "@config/centraldb/postgresql.conf|config/centraldb/postgresql.conf.default"
+      settings:
+        maxConnections: 100
```

Comment:
The previous `postgresql.conf.default` set this to 200. Is it intentional to reduce maxConnections to 100?

Reply:
yes that was intentional.  the default connection string itself limited it to 100 anyway and this database only supports the central connection.